### ISSUE TITLE
Self-hosted AWS S3 storage, e.g Minio

### ISSUE THREAD

**Issue by 917huB** [standardnotes/syncing-server#110]
Is it possible to configure fileSafe to use self hosted AWS S3 compatible storage provisioned through services like Minio?

**Comment by mobitar** [comment]
No, FileSafe only works with literal S3 for now, and not S3 compatible APIs.
